### DISCUSSION TITLE
GraphMultisetTransformer with conv that is none of GCNConv, GraphConv or GATConv

### DISCUSSION THREAD

**Discussion by ldv1** [pyg-team/pytorch_geometric#5368] [General]
Hi,
According to the documentation, the [GraphMultisetTransformer](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.aggr.GraphMultisetTransformer) accepts the option `conv` that must be one of `GCNConv`, `GraphConv` or `GATConv`. I have skimmed through the code, and it seems to me that any other type of graph convolutions could be accepted, as long as they do not need edge attributes. At least, there is no check in the code that it must be so.
Do I err ? Maybe those are the convolutions that were tested in the paper.
Thanks for any help.

**Comment by rusty1s** [comment]
Yes, I don‘t think there is a limitation on what GNN you can use. These are just the ones we tested against.

**Reply by ldv1** [reply]
Thanks for the prompt answer.
